app/build.gradle

```diff
@@ -2,13 +2,13 @@ apply plugin: 'com.android.application'
 apply plugin: 'com.moowork.node'
 
 android {
-    compileSdkVersion 22
+    compileSdkVersion 23
     buildToolsVersion "21.1.2"
 
     defaultConfig {
         applicationId "org.mozilla.webmaker"
         minSdkVersion 14
-        targetSdkVersion 22
+        targetSdkVersion 23
```

Comment:
Its usually a good idea to always target the latest version of Android, whilst still providing support down to the lowest point possible (SDK 14), I had to update this because the dependency updates below were complaining rather loudly about it.